packages/react-native-bpk-component-navigation-bar/src/isIphoneX.js

```diff
@@ -17,9 +17,13 @@
  */
 /* @flow */
 
-import { DeviceInfo } from 'react-native';
+import { DeviceInfo, Platform } from 'react-native';
 
 export default (() => {
+  if (Platform.OS === 'android') {
+    return false;
+  }
+
```

Comment:
Importing this in stories was crashing Android so this seemed like the easiest fix. An alternative approach would be to rename this file `isIphoneX.ios.js` and then having a `.android.js` one that just returned false, but that feels like overengineering.